samples/csharp_dotnetcore/83.openai/appsettings.json

```diff
@@ -0,0 +1,10 @@
+{
+  "MicrosoftAppType": "",
+  "MicrosoftAppId": "",
+  "MicrosoftAppPassword": "",
+  "MicrosoftAppTenantId": "",
+  "OpenAIKey": "",

```

Comment:
If I fill in both open ai key and azure ai key, which one will be used?

Reply:
Both are used